### PR TITLE
 Fix bugs in Intl

### PR DESCRIPTION
**Pull request by ksh8281**

### CODE REVIEW

File: src/runtime/StaticStrings.h [hunk 1]
```diff
@@ -419,7 +419,8 @@ namespace Escargot {
     F(trimLeft)                   \
     F(asyncIterator)              \
     F(AsyncGenerator)             \
-    F(AsyncGeneratorFunction)
+    F(AsyncGeneratorFunction)     \
+    F(usage)
```
Comment (clover2123) [inline src/runtime/StaticStrings.h:423]: It seems that `usage` is nowhere used.

File: src/runtime/Object.h [hunk 2]
```diff
@@ -722,6 +722,9 @@ class Object : public PointerValue {
     explicit Object(ExecutionState& state);
     explicit Object(ExecutionState& state, Object* proto);
 
+    enum PrototypeIsNullTag { PrototypeIsNull };
+    explicit Object(ExecutionState& state, PrototypeIsNullTag); // I added new function for reducing checking null for prototype
```
Comment (clover2123) [inline src/runtime/Object.h:726]: How about adding "__" prefix and postfix for `PrototypeIsNull`?
Other enum values such as ForGlobalBuiltin, ForBuiltinConstructor use the same prefix and postfix.

Comment (ksh8281) [reply src/runtime/Object.h:726]: IMO adding __ at prefix, postfix means `the flag or something` is internal or users would not use that flag or something.
but I want to encourage using this function.
I think we can move __ForGlobalBuiltin__ into private area.

File: src/runtime/StaticStrings.h (hunk 1 above)
Comment (ksh8281) [reply src/runtime/StaticStrings.h:423]: Oops my fault

File: src/runtime/Object.h (hunk 2 above)
Comment (clover2123) [reply src/runtime/Object.h:724]: Great idea!

**Pull request merged**